main-app/tests/lobby-controller.test.js

```diff
@@ -0,0 +1,68 @@
+(function () {
+    'use strict';
+
+    describe('lobby-controller tests', function(){
+        var $scope, $rootScope, controller, mySpy, mySpy2, mySpy3, $q, sandbox;
```

Comment:
Poor naming on the spies - name them after the objects/functions they are testing.